x-pack/plugins/apm/server/routes/assistant_functions/route.ts

```diff
@@ -260,6 +260,7 @@ const getApmServicesListRoute = createApmServerRoute({
       rollupInterval: RollupInterval.OneMinute,
       serviceGroup: null,
       mlClient,
+      useDurationSummary: true,
```

Comment:
Perhaps safer to leave this to `false`. I'll let @dgieselaar decide though

Reply:
Given its using Transaction Metrics, i believe this should work just fine. But i will wait for Darios comment

Reply:
Agree with Søren that this is probably better off as `false`. I think we can safely assume transaction metrics are available given they have been since 7.14, but `transaction.duration.summary` is fairly recent, and we don't do a check here to see if it exists at all.